CHANGELOG.rst

```diff
@@ -1,67 +1,66 @@
 ********************
-[0.2.0] - XXXX-XX-XX
+[0.2.0] - 2020-12-18
 ********************
 
+Major feature release, including some incompatible file format and API updates.
+
 **New features**:
 
+- Mismatch and recombination parameters can now be specified via the
+  recombination_rate and mismatch arguments in the Python API.
+
+- Missing data can be accomodated in SampleData using the tskit.MISSING_DATA
+  value in input genotypes.
+
+- Metadata schemas for population, individual, site and tree sequence metadata
+  can now we be specified in the SampleData format. These will be included
+  in the final tree sequence and allow for automatic decoding of JSON metadata.
```

Comment:
Do we want to say that we add metadata to sites & individuals when doing inference? I guess it's not that important for most people

Reply:
I think that's the role of documentation, this is just to let people know what changed.